save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/demo/welcome/DemoListComponent.kt

```diff
@@ -26,7 +26,7 @@ import kotlinx.serialization.json.Json
 
 private val withBackground = cardComponent(isBordered = true, hasBg = true, isPaddingBottomNull = true)
 
-val demoList = FC<Props> {
+val demoList: FC<Props> = FC {
```

Comment:
```suggestion
val demoList: FC = FC {
```

Can't you do it so?

```Kotlin
val demoList = FC {
```

or even so?

Reply:
I can't do `val demoList: FC = FC {` because `FC` requires a generic for a props.

This one `val demoList = FC<T> {` is not valid by diktat style

Reply:
Fun thing is that this seems to work (tested with `1.0.0-pre.598`):
```kotlin
@JsPreview
val test = FC {
    div {
        +"testing"
    }
}
```

According to [sources](https://github.com/JetBrains/kotlin-wrappers/blob/fd77761f4ed7687692bdd58182f65617563192e3/kotlin-react/src/jsMain/kotlin/react/FC.kt#L3) there is a function `FC` that does not have any generic so it returns `FC<Props>`, which is quite convenient

Reply:
will try this one `val test = FC {`

Reply:
done

Reply:
looks like it's possible because of https://github.com/JetBrains/kotlin-wrappers/blob/def862d27744f4b0c8cea95df6e4e3274f8a282c/kotlin-react/src/jsMain/kotlin/react/FC.kt#L3C1-L10C2

cannot do it in current version :(